src/api/participants/handler.js

```diff
@@ -24,7 +24,7 @@
 
 'use strict'
 
-const Participant = require('../../domain/participant')
+const ParticipantService = require('../../domain/participant')
```

Comment:
Just for interest....are we using '`{Domain-Entity}`**-Service**' as a naming convention?

Reply:
Yes, we have:
'`{Domain-Entity}`**-Service**'
'`{Domain-Entity}`**-Model**'
'`{Domain-Entity}`**-Facade**',
but it seems there are some files which need updating.